tests/admin_views/test_multidb.py

```diff
@@ -0,0 +1,45 @@
+from django.conf.urls import url
+from django.contrib import admin
+from django.contrib.auth.models import User
+from django.db import connections
+from django.test import TestCase, mock, override_settings
+
+from .models import Book
+
+
+class Router(object):
+    target_db = None
+
+    def db_for_read(self, model, **hints):
+        return self.target_db
+
+    db_for_write = db_for_read
+
+site = admin.AdminSite(name='test_adminsite')
+site.register(Book)
+
+urlpatterns = [
+    url(r'^admin/', site.urls),
+]
+
+
+@override_settings(ROOT_URLCONF=__name__, DATABASE_ROUTERS=['admin_views.test_multidb.Router'])
```

Comment:
`['%s.Router' % __name__]`
And could you call rename this file to `test_admin_multidb.py`?